app/styles/components/c-button.scss

```diff
@@ -34,3 +34,18 @@
   text-transform: uppercase;
   width: 100%;
 }
+
+.c-button-submit-small {
+  background-color: $color-3;
+  border: 0;
+  color: $color-4;
+  display: block;
+  font-size: $font-size-small;
+  letter-spacing: -.1px;
+  margin: 45px auto 0;
+  max-width: 115px;
+  outline: 0;
+  padding: 10px 0;
+  text-transform: uppercase;
+  width: 100%;
+}
```

Comment:
Uhm.. This is not the idea of a css component style. There must be ONE global component with basic and common styles (try to avoid positioning and sizing). Then, use modifiers, inside the component, to get different types of buttons if need it.

Let me know if you need help this. This is important and should be clear, ok? If you need it, we can talk about it.